accesspoc/dips/views.py

```diff
@@ -605,3 +614,18 @@ def download_dip(request, pk):
         return response
     except FileNotFoundError:
         raise Http404('ZIP file not found.')
+
+
+@login_required(login_url='/login/')
+def settings(request):
+    # Only admins can manage settings
+    if not request.user.is_superuser:
+        return redirect('home')
+
+    form = DublinCoreSettingsForm(request.POST or None)
+    if request.method == 'POST' and form.is_valid():
+        form.save()
+        # Redirect to not add validation classes
```

Comment:
What does this mean?

What would happen if the form is received but it's invalid? Shouldn't we be showing a validation error?